arch/risc-v/src/qemu-rv32/qemu_rv32_head.S

```diff
@@ -105,21 +105,6 @@ exception_common:
   sw   x30, 30*4(sp)  /* t5 */
   sw   x31, 31*4(sp)  /* t6 */
 
-#if defined(INT_XCPT_REGS) && INT_XCPT_REGS >= 39
-  csrr x28, 0x7b0
-  csrr x29, 0x7b1
-  csrr x30, 0x7b2
-  sw x28, 33*4(sp)
-  sw x29, 34*4(sp)
-  sw x30, 35*4(sp)
-  csrr x28, 0x7b4
-  csrr x29, 0x7b5
-  csrr x30, 0x7b6
-  sw x28, 36*4(sp)
-  sw x29, 37*4(sp)
-  sw x30, 38*4(sp)
-#endif
-
```

Comment:
Why this is removed?

Reply:
It's only works with RV32M1 chips with customized toolchain,